README.md

```diff
@@ -62,6 +65,7 @@ const packmap = require("packmap");
 const options = {
   outdir: "dist", // defaults to dist
   package: "path/to/package.json" // defaults to package.json
```

Comment:
I think we're missing a comma here?

Reply:
Good catch -- at first I thought this was in the code itself and was worried we published a bad version. But this is the readme and is only documentation. I'll send a PR to fix.

Reply:
fixed in https://github.com/openmrs/packmap/commit/b06133a9c38ebeb7484c7a6c8a71e69542156e4c